### PR TITLE
update patch doc to 5_0_2_1

### PR DESCRIPTION
**Pull request by stephenwaite**

### CODE REVIEW

**Comment by stephenwaite** [comment]
found a cool site to check out scripts, https://www.shellcheck.net/

File: docker/mysql-xtrabackup/Dockerfile [hunk 1]
```diff
@@ -42,7 +42,7 @@ RUN set -ex; \
 	apt-key list > /dev/null
 
 ENV MYSQL_MAJOR 5.7
-ENV MYSQL_VERSION 5.7.27-1debian8
+ENV MYSQL_VERSION 5.7.28-1debian8
```
Comment (bradymiller) [inline docker/mysql-xtrabackup/Dockerfile]: hi @jesdynf ,
Let me know if you need me to rebuild this docker with this change (when Stephen brings it in) and push it to dockerhub.
thanks, -brady

Comment (stephenwaite) [reply docker/mysql-xtrabackup/Dockerfile]: hi @bradymiller , don't have write access to this repo (for a good reason 😄)

**Comment by bradymiller** [comment]
hi @stephenwaite , Bringing this in.Thanks! -brady

**Pull request merged**